.gitignore

```diff
@@ -78,4 +78,8 @@ credentials
 
 
 
-public/js/dist
\ No newline at end of file
+public/js/dist
+
+# Will uncomment when we are setting up langauge keys
```

Comment:
Can remove this line

```suggestion
# Will uncomment when we are setting up langauge keys
```

Reply:
Yes I'll also remove the changes to the .gitignore for now so we either stop tracking the files i.e. delete them or we just check in all of the language strings